src/anomalib/utils/metrics/pro.py

```diff
@@ -38,10 +38,12 @@ def compute(self) -> Tensor:
         target = dim_zero_cat(self.target)
         preds = dim_zero_cat(self.preds)
 
+        target = target.unsqueeze(1)  # kornia expects N1HW format
+        target = target.type(torch.float)  # kornia expects FloatTensor
```

Comment:
Just to make it more Pythonic and idiomatic...
```suggestion
        target = target.unsqueeze(1).type(torch.float)  # kornia expects N1HW and FloatTensor format
```

Reply:
Okay. I just wrote it as such since it's like that in aupro implementation. But I agree this is better :).